src/main/java/codeu/controller/ProfileListServlet.java

```diff
@@ -0,0 +1,61 @@
+package codeu.controller;
+
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.User;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created by gavinlifrieri on 6/12/18.
```

Comment:
Maybe add a comment explaining what this servlet does? Stuff like which URL it's mapped to or what data it serves is probably pretty useful.